From: Michael Chan <michael.chan@broadcom.com>
Date: Sun, 15 May 2016 03:04:48 -0400
Subject: bnxt_en: Fix length value in dmesg log firmware error message.
Patch-mainline: v4.7-rc1
Git-commit: 8578d6c19a308dea3daf3d03acdf18724ec05590
References: bsc#963575 FATE#320144

The len value in the hwrm error message is wrong.  Use the properly adjusted
value in the variable len.

Signed-off-by: Michael Chan <michael.chan@broadcom.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 drivers/net/ethernet/broadcom/bnxt/bnxt.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/broadcom/bnxt/bnxt.c
+++ b/drivers/net/ethernet/broadcom/bnxt/bnxt.c
@@ -2836,7 +2836,7 @@ static int bnxt_hwrm_do_send_msg(struct
 		if (i >= tmo_count) {
 			netdev_err(bp->dev, "Error (timeout: %d) msg {0x%x 0x%x} len:%d\n",
 				   timeout, le16_to_cpu(req->req_type),
-				   le16_to_cpu(req->seq_id), *resp_len);
+				   le16_to_cpu(req->seq_id), len);
 			return -1;
 		}
 
